release notes/v0.52.0.md

```diff
@@ -0,0 +1,148 @@
+k6 `v0.52.0` is here 🎉! Some special mentions included in this release:
+
+- [We switched from `goja` to our own fork named `sobek`](#switch-goja-to-our-own-fork-named-sobek-3775)
+- [Panics are no longer being captured](#panics-are-no-longer-being-captured-3777)
+- [The `web-dashboard` built-in output becomes `dashboard`](#the-web-dashboard-built-in-output-becomes-dashboard-xk6-dashboard163)
+
+## Breaking changes
+
+### Switch `goja` to our own fork named `sobek` [#3775](https://github.com/grafana/k6/pull/3775)
+
+In order to accelerate the development speed and unblock the support for [Native ESM](https://github.com/grafana/k6/issues/3265),
+we have decided to create a fork of the [`goja`](https://github.com/dop251/goja/) project under the Grafana GitHub organization,
+named [`sobek`](https://github.com/grafana/sobek).
+
+So, since this release, k6 (and its extensions) starts using `sobek` instead of the original `goja`, for all (of the
+publicly exposed parts of the API) except for a couple of packages that are only used internally by k6.
+
+Find further details in [#3772](https://github.com/grafana/k6/issues/3772) and [#3773](https://github.com/grafana/k6/issues/3773).
```

Comment:
Probably, we should mention that it mainly affects the extensions, and we might need to mention the migration steps to simplify the life of extension developers. @mstoykov WDYT?

Reply:
see https://github.com/grafana/k6/pull/3805/files#r1648494026